src/pages/iou/request/IOURequestRedirectToStartPage.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React, {useEffect} from 'react';
+import _ from 'underscore';
```

Comment:
We prefer lodash to underscore.
Btw not blocker since both will be deprecated during TS migration